docsets/Solarus/docset.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Solarus",
+    "version": "1.6.0",
+    "archive": "Solarus.docset",
```

Comment:
This needs to be a `tgz` and it needs to exist. It's not enough to have the archive only in `versions`.

Reply:
hopefully, it's fine now, thanks.